src/backend/Node.js

```diff
@@ -15,3 +15,5 @@ class Node {
     }
   }
 }
+
+module.exports = Node;
```

Comment:
This will be fun.... Jon made the same change lol. Whichever of you merges first, the other should pull and deal with that lol